politeiawww/inventory.go

```diff
@@ -379,6 +419,74 @@ func (b *backend) loadComments(t string) error {
 	return nil
 }
 
+// loadComments calls out to the decred plugin to obtain all comments likes
+// of a given proposal
+//
+// This function must be called WITH the mutex held.
+func (b *backend) loadCommentsLikes(token string) error {
```

Comment:
If this needs to be called with the mutex held it should be named `_loadCommentsLikes`, right?